### PR TITLE
Donut hours fix

### PR DESCRIPTION
**Pull request by brettjforsyth**
For #501. Fixes some rounding issues on the donut causing the hours not to display properly.

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -1122,10 +1123,13 @@ function eypd_d3_array( $data ) {
...
 
 		}
+		if ( 0 === strcmp( 'Professional Development Certificate Credit Hours', $key ) ) {
+			$total = $total + intval( $event['hours'] );
+		}
 
 		foreach ( $cat as $k => $v ) {
 			$result[ $i ]['label'] = html_entity_decode( $k );
-			$result[ $i ]['value'] = number_format( $v['value'], 1 );
+			$result[ $i ]['value'] = number_format( $v['value'], 8 );
```
Comment (bdolor) [inline functions.php]: assuming `8` is not intentional? 
image

Comment (bdolor) [reply functions.php]: hold on...it's me.

File: functions.php [hunk 2]
```diff
@@ -1122,10 +1123,13 @@ function eypd_d3_array( $data ) {
 			unset( $unit );
 
 		}
+		if ( 0 === strcmp( 'Professional Development Certificate Credit Hours', $key ) ) {
```
Comment (bdolor) [inline functions.php]: `$key` is undefined at this point, as is `$total`, which is never referenced again - can you indicate what purpose this serves?

Comment (brettjforsyth) [reply functions.php]: abandoned train of thought that I didn't remove. 1104 can go and 1126-1128

**Pull request merged**